integtests/IntegTests/HBSeriesDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntegTests
+{
+    /// <summary>
+    /// Class representing a Series
+    /// </summary>
+    class HBSeriesDto
```

Comment:
Why create this class ? You cannot use the one from metadatadatbase ?

Reply:
IMHO it is not duplication, it is decoupling.

Reply:
Ok